ui-tests/src/test/java/io/syndesis/qe/pages/connections/fragments/list/ConnectionsList.java

```diff
@@ -18,7 +18,7 @@
 
     private static final class Element {
         public static final By TECH_PREVIEW = By.xpath("syndesis-card-tech-preview");
-        public static final String CONNECTION_CARD = "div[data-testid=\"connection-card-%s-card\"]";
+        public static final String CONNECTION_CARD = "article[data-testid=\"connection-card-%s-card\"]";
```

Comment:
What about we use `*` instead of specific tag type to avoid future failures?

Reply:
We can. The element is determined by the testid anyway.